console/src/main/webapp/manager/app/components/area/area.es6

```diff
@@ -97,10 +97,17 @@ class AreaController {
       vector.getSource().addFeatures(format.readFeatures(json, conf))
       vector.getSource().forEachFeature(f => {
         const group = f.get(config.areas.group)
+        if (group === undefined) {
+          throw new Error(`Cannot get AreaGroup ${config.areas.group} in provided geojson. Check datadir config.`)
+        }
         if (this.groups.indexOf(group) < 0) {
           this.groups.push(group)
         }
-        f.set('_label', f.get(config.areas.value).toString() || '')
+        const displayName = f.get(config.areas.value)
+        if (displayName === undefined) {
+          throw new Error(`Cannot get AreaValue ${config.areas.value} in provided geojson. Check datadir config.`)
```

Comment:
```suggestion
          throw new Error(`Cannot get AreaValue «${config.areas.value}» in provided geojson. Check datadir config.`)
```

Reply:
Sorry but « and » are for French. If English I'll use double quote :-)